application/common/nhs.py

```diff
@@ -229,7 +229,8 @@ def is_spec_opening_json(item: dict) -> bool:
 
 
 def match_nhs_entities_to_services(
-        nhs_entities: List[NHSEntity], services: List[DoSService]) -> Dict[str, List[DoSService]]:
+    nhs_entities: List[NHSEntity], services: List[DoSService]
```

Comment:
The formatting on these function parameters seems a bit strange

Reply:
Yes, just tried to change it locally and the formatting changed it back. Must be due to line length.

Reply:
ok no worries just want to check